tests/lib/pdn/src/main.c

```diff
@@ -1235,6 +1235,13 @@ void test_pdn_deactivate(void)
 	TEST_ASSERT_EQUAL(PDN_EVENT_DEACTIVATED, pdn_evt_event);
 }
 
+void test_detach(void)
+{
+	on_cgev("+CGEV: NW DETACH\r\n");
+
+	TEST_ASSERT_EQUAL(CID_1, pdn_evt_cid);
```

Comment:
So we are not testing with a list of PDNs here? Would be nice but would also need a bit of work for the test "setup".